src/otx/core/model/entity/detection.py

```diff
@@ -327,7 +330,19 @@ def __init__(
             max_num_requests,
             use_throughput_mode,
             model_api_configuration,
+            tile_config,
         )
+        if self.tile_config is not None and self.tile_config.enable_tiler:
+            log.info(
+                f"Enable tiler with tile size: {self.tile_config.tile_size} and overlap: {self.tile_config.overlap}",
+            )
+            ov_tile_config = {
+                "tile_size": self.tile_config.tile_size[0],
```

Comment:
Are these parameters also added to the model meta info on export?
Other MAPI clients can rely only on information coming from .xml